R/merge.R

```diff
@@ -188,24 +188,24 @@
     seq
 }
 
-.merge_rows_TSE <- function(x, f, archetype = 1L, mergeTree = FALSE,
-                            mergeRefSeq = FALSE, ...){
+.merge_rows_TSE <- function(x, f, archetype = 1L, update.tree = FALSE,
+                            update.refseq = FALSE, ...){
     # input check
-    if(!.is_a_bool(mergeTree)){
```

Comment:
Check that agglomerateByRank works also with

agglomerateByRank(tse, mergeRefSeq=TRUE)

Reply:
It seems that there is no support for old values, but I am not sure since this is just internal function

Reply:
Yes i didn't support these ones because they were internal. should add support?

Reply:
If these work

```
agglomerateByRank(tse, agglomerate.tree = TRUE, mergeRefSeq=TRUE)
agglomerateByVariable(tse, MARGIN = 1, mergeRowTree = TRUE, mergeRefSeq=TRUE)
```

then no. If they do not work, then you have to add support

Reply:
The functionality should stay the same for user

Reply:
Functionality restored for user.